gapis/api/vulkan/read_framebuffer.go

```diff
@@ -152,7 +152,7 @@ func (t *readFramebuffer) Depth(ctx context.Context, id api.SubCmdIdx, idx uint3
 			// first one.
 			// TODO: support multi-layer rendering.
 			layer := imageViewDepth.SubresourceRange().BaseArrayLayer()
-			return t.postImageData(ctx, cb, id, s, cmd.cmdBuffer, cmd.pendingCommandBuffers, depthImageObject, imageViewDepth.Fmt(), VkImageAspectFlagBits_VK_IMAGE_ASPECT_DEPTH_BIT, layer, level, w, h, w, h, out, res)
+			return t.postImageData(ctx, cb, id, s, cmd.cmdBuffer, cmd.pendingCommandBuffers, depthImageObject, imageViewDepth.Fmt(), VkImageAspectFlagBits_VK_IMAGE_ASPECT_DEPTH_BIT, layer, level, w, h, width, height, out, res)
```

Comment:
Since the two pairs of size params of this function are

```
func (t *readFramebuffer) postImageData(ctx context.Context,
...
	imgWidth,
	imgHeight,
	requestWidth,
	requestHeight uint32,
...
```
could you please rename the ```w,h``` and ```width, height``` accordingly, for clarify purposes?

Reply:
I'm ok if you want to leave `w` and `h` as is, to not pollute the change. But you could use `req[uest]Width` and `req[uest]Height` for the new parameters.